### ISSUE TITLE
Inconsistent Messaging Groups Ids in "Take Back the Shelves" campaign

### ISSUE THREAD

**Issue by sergii-tkachenko** [DoSomething/gambit#747]
### Current behavior
Both production and staging (thor) have same data in `mobilecommons_group_*` groups:

[Thor](http://ds-mdata-responder-staging.herokuapp.com/v1/campaigns/7277):
```json
{
  "data": {
    "id": 7277,
    "title": "Take Back the Shelves",
    "campaignbot": true,
    "status": "active",
    "current_run": 7317,
    "mobilecommons_group_doing": 258145,
    "mobilecommons_group_completed": 258166,
    "keywords": [
      "SHELVESBOT"
    ]
  }
}
```

[Prod](http://ds-mdata-responder.herokuapp.com/v1/campaigns/7277):
```json
{
  "data": {
    "id": 7277,
    "title": "Take Back the Shelves",
    "campaignbot": true,
    "status": "active",
    "current_run": 7279,
    "mobilecommons_group_doing": 258145,
    "mobilecommons_group_completed": 258166,
    "keywords": [
      "SHELVES"
    ]
  }
}
```

Mobile commons group [258145](https://secure.mcommons.com/groups/258145) title says group belongs to thor, run_id `7317`: `campaign_id=7277 run_id=7317 field=doing env=thor`. Same goes for group [258166](https://secure.mcommons.com/groups/258166): `campaign_id=7277 run_id=7317 field=completed env=thor`,

### Desired behavior
Production `mobilecommons_group_doing` and `mobilecommons_group_completed` for "Take Back the Shelves" should be different from Thor and have run id `7279` assigned.

cc @deadlybutter

**Comment by sergii-tkachenko** [comment]
Apparently, there was no connection between Gambit and Messaging Groups API.
Messaging Groups API was waiting for `x-gambit-group-api-key:` header but Gambit was sending `x-messaging-group-api-key:`. @deadlybutter will change Messaging Groups API to accept `x-messaging-group-api-key` header.

**Comment by deadlybutter** [comment]
Fixed here: https://github.com/DoSomething/messaging-groups-api/commit/41973bd27247311c131aad5a09717cbe2965bc5f

**Comment by sergii-tkachenko** [comment]
More, this particular campaign is not available on Messaging Groups API:

```
curl -H "x-messaging-group-api-key: REDACTED" -H "Accept: application/json" https://messaging-groups-prod.herokuapp.com/api/v1/group/7277/7279
No group for this campaign id & run id exists.%
```

**Comment by sergii-tkachenko** [comment]
Fixed in #750.

New groups are created for run id 7279:
```
2016-11-17T20:43:32.130034+00:00 app[web.1]: [32minfo[39m: Setting messaging groups: campaign 7277 run 7279
2016-11-17T20:43:33.033725+00:00 app[web.1]: [32minfo[39m: Messaging Groups created: campaign 7277 run 7279: 263380 / 263383
```

**Comment by sergii-tkachenko** [comment]
I will manually move users from
- 258145 to 263380
- 258166 to 263383
